### ISSUE TITLE
Issues discovered late during Conway development

### ISSUE THREAD

**Issue by WhatisRT** [IntersectMBO/formal-ledger-specifications#570]
This ticket is for keeping track of issues discovered or fixed recently.

- #490 (Done, waiting on conformance)
- #491 (WIP, estimate: 2 days)
- #508
- #510
- #511 (estimate: 3 days)
- #512 (WIP, estimate: 1 week)
- #537
- #538 (Done, waiting on conformance)
- #539
- #540
- #541
- #545 (not super important, estimate: 2 days)
- #571 (done)
- #578

**Comment by jmchapman** [comment]
@williamdemeo can you add the tickets you're working on as a comment and perhaps they should added to this list

**Comment by williamdemeo** [comment]
@jmchapman 

Lately I've been working on #467, which is not on this list, but Andre requested I work on that after finishing #537.

As for the issues on this list:

- #512. I worked on it for a while, but we agreed to put that on hold for now (reasons given in the comments on the issue page).

- #537.  Completed.  My PR was awaiting a second review for about a week and then it broke when I merged the latest master.  I will fix that today and ping @WhatisRT on slack to see if he can't review it.

- #538.  Completed.  (Not merged yet---waiting on conformance.)

Andre has completed #490 (though the PR has fallen behind master), and it seems he is working on #491.

That leaves the following: #511 #545 #571.

I agreed to work on #571 as Alexi asked me to add it to my backlog.  I plan to work on it as soon as I get #467 closer to completion.
